content/javascript/concepts/dates/terms/getUTCMilliseconds/getUTCMilliseconds.md

```diff
@@ -0,0 +1,39 @@
+---
+Title: 'getUTCMilliseconds()' # Required; the file name should be the same as the title, but lowercase, with dashes instead of spaces, and all punctuation removed
```

Comment:
```suggestion
Title: '.getUTCMilliseconds()' # Required; the file name should be the same as the title, but lowercase, with dashes instead of spaces, and all punctuation removed
```